### ISSUE TITLE
Bring back copier bot

### ISSUE THREAD

**Issue by MridulS** [scipp/copier_template#175]
We should revert https://github.com/scipp/copier_template/pull/171 once things have settled down with all the ruff shenanigans.

**Comment by YooSunYoung** [comment]
I thought we are going to deploy a ``scipp-bot`` will we still need the `copier.yml`...?

**Comment by MridulS** [comment]
Yes, the instructions about what to do will be in the copier.yml file. The bot account will just be the actor of the commits and pushes.

**Comment by YooSunYoung** [comment]
Can we make the bot in a way that it just opens the PR with merge conflict so that reviewers can resolve it manually...?

**Comment by MridulS** [comment]
Yes, the last iteration created a `.rej` file with the conflicts, we can just inline them (the default behaviour).

**Comment by YooSunYoung** [comment]
`essnmx` seems to be outdated so we can try there.
